xrpl/asyncio/wallet/wallet_generation.py

```diff
@@ -13,9 +13,9 @@
 _TEST_FAUCET_URL: Final[str] = "https://faucet.altnet.rippletest.net/accounts"
 _DEV_FAUCET_URL: Final[str] = "https://faucet.devnet.rippletest.net/accounts"
 _AMM_DEV_FAUCET_URL: Final[str] = "https://ammfaucet.devnet.rippletest.net/accounts"
-_HOOKS_V2_TEST_FAUCET_URL: Final[
+_HOOKS_V3_TEST_FAUCET_URL: Final[
     str
-] = "https://hooks-testnet-v2.xrpl-labs.com/accounts"
+] = "https://hooks-testnet-v3.xrpl-labs.com/accounts"
```

Comment:
These changes were in another PR.  Maybe the branch `xrpl-py-2.0` needs to be rebased with `master`.